### PR TITLE
LPD-19497 the friendly url must return a value for the default language not the site default language

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
Steps to reproduce:

Execute the JournalArticleFriendlyURLTest

- **LPD-19497 when adding the journal article on the test allow to set the default language**
- **LPD-19497 add test to check that the site default language is not generated if the default language is different on the creation of the**
- **LPD-19497 the default locale should be the one that not can be null**
- **LPD-19497 autoSF**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:friendly-url

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2546666788

File: modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/friendly/url/test/JournalArticleFriendlyURLTest.java [hunk 1]
```diff
@@ -134,6 +135,27 @@ public void testFriendlyURLAfterUpdate() throws Exception {
 		Assert.assertEquals(friendlyURLMap, updatedArticle.getFriendlyURLMap());
 	}
 
+	@Test
+	public void testFriendlyURLShouldNotHaveSiteLanguageNotEqualDefaultLanguage()
+		throws Exception {
+
+		LocaleThreadLocal.setSiteDefaultLocale(LocaleUtil.US);
```
Comment (jkappler) [inline modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/friendly/url/test/JournalArticleFriendlyURLTest.java:142]: Hi @AliciaGarciaGarcia, all looks great! Just the comment that we talked about. Would it make sense set the current value again after the test finishes in the thread local?

Comment (AliciaGarciaGarcia) [reply modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/friendly/url/test/JournalArticleFriendlyURLTest.java:142]: Thanks, I will send an additional commit.

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1397401_9629

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2547498001

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:report:2547570223

**Comment by jkappler** [comment]
ci:forward:force

**Comment by AliciaGarciaGarcia** [comment]
ci:forward:force

**Comment by AliciaGarciaGarcia** [comment]
ci:forward:force

**Comment by jkappler** [comment]
ci:forward:force

**Comment by AliciaGarciaGarcia** [comment]
this is strange

**Comment by AliciaGarciaGarcia** [comment]
sent to brian https://github.com/brianchandotcom/liferay-portal/pull/147968

**Pull request closed without merging**